Lib/test/_test_multiprocessing.py

```diff
@@ -98,6 +98,11 @@ def wait_for_handle(handle, timeout):
 except:
     MAXFD = 256
 
+# Particular buildbots can be exceptionally slow.  Rather than raise timeouts
+# (necessary in some tests) across the board to accomodate them, sensible
+# timeout values may be scaled by this multiplier on such a buildbot.
+TIMEOUT_MULTIPLIER = float(os.getenv("CONF_TIMEOUT_MULTIPLIER", default=1))
```

Comment:
I would prefer to see this constant in test.support. I also suggest to rename it to TEST_TIMEOUT_MULTIPLIER or TEST_TIMEOUT_FACTOR.